docs/node/run-your-node/rofl-node.mdx

```diff
@@ -0,0 +1,332 @@
+import DocCard from '@theme/DocCard';
+import {findSidebarItem} from '@site/src/sidebarUtils';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# ROFL Node
+
+:::info
+
+These instructions are for setting up a _ROFL node_ which executes ROFLs inside
+a TEE, but otherwise only observes the ParaTime activity and can also submit
+transactions.
+
+:::
+
+This guide will cover setting up your ROFL node for running ROFL apps on the
+Oasis Network. This guide assumes some basic knowledge on the use of command
+line tools.
+
+## Prerequisites
+
+### ParaTime Client Node
+
+ROFL node is a special kind of a **ParaTime Client Node** with a TEE-capable
+hardware in order to securely run ROFLs. First, complete the ParaTime Client
+Node instructions:
+
+1. Set up your [ParaTime Client Node] including support for [TEE ParaTimes].
+2. Add the Sapphire ParaTime to your node ([Mainnet], [Testnet]).
```

Comment:
Btw, we need to update documentation as one no longer need sto manually download ORCs. But that is a separate issue.